materials/dplyr.md

```diff
@@ -10,26 +10,25 @@ language: R
 > * download [`surveys.csv`](https://ndownloader.figshare.com/files/2292172).
 > * download [`species.csv`](https://ndownloader.figshare.com/files/3299483).
 > * download [`portal_mammals.sqlite`](https://ndownloader.figshare.com/files/2292171).
-
+>
+> * Consider removing the `dplyr` package so you can demonstrate installing it.
+>     * Linux users: you may not want to do this because the source install is slow
 
 ### Packages
 
 * Main way that reusable code is shared in R
 * Combination of code, data, and documentation
+* R has a rich ecosystem of packages for data manipulation & analysis
 * Download and install packages with the R console:
     * `install.packages("dplyr")`
-* Using a package:
-    * Load all of the functions in the package: `library("dplyr")`
```

Comment:
I would think that it would be important to introduce students to this material, at least `library(dplyr)`. Does it belong somewhere else?